ecf/scripts/gdas/atmos/analysis/jgdas_atmos_analysis.ecf

```diff
@@ -29,21 +21,21 @@ export model=gfs
 ############################################################
 # Load modules
 ############################################################
-module load lsf/${lsf_ver}
-module load impi/${impi_ver}
-module load CFP/${CFP_ver}
+module load cray-mpich/${cray_mpich_ver}
+module load cray-pals/${cray_pals_ver}
+module load cfp/${cfp_ver}
 module load python/${python_ver}
-module load NetCDF-parallel/${netcdf_parallel_ver}
-module load HDF5-parallel/${hdf5_parallel_ver}
-module load crtm/${crtm_ver}
+module load libjpeg/${libjpeg_ver}
+module load hdf5/${hdf5_ver}
+module load netcdf/${netcdf_ver}
 module load grib_util/${grib_util_ver}
+module load crtm/${crtm_ver}
 
 module list
 
 #############################################################
 # WCOSS environment settings
 #############################################################
-export KMP_AFFINITY=scatter
 export OMP_NUM_THREADS=$threads
```

Comment:
Where is the value for `$threads` coming from?  It was previously set on line 20 above.

Reply:
Removed.